CTFd/utils/__init__.py

```diff
@@ -28,20 +22,12 @@ def _get_config(key):
         if value and value.isdigit():
             return int(value)
         elif value and isinstance(value, six.string_types):
-            if value.lower() == 'true':
-                return True
-            elif value.lower() == 'false':
-                return False
-            else:
-                return value
+            return {'true': True, 'false', False}.get(value.lower(), value)
```

Comment:
SyntaxError here causing your test failures I believe. I don't see this as more readable, is there some gain to be had here besides the condensed code? If not please revert this change.